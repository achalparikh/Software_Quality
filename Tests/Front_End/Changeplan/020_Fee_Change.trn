10                      00000 00000000 A 
08 John Doe             00000 00000000   
00                      00000 00000000   
10 John Doe             00000 00000000   
01 John Doe             00000 00500.00 S 
00                      00000 00000000   