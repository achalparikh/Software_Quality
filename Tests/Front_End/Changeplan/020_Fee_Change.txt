login
admin
changeplan
John Doe
00012
logout
login
standard
John Doe
withdraw
00012
500.00
logout
quit