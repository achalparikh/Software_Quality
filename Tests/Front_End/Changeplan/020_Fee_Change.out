Login to start:
Enter type of account: "standard" or "admin"
You are now logged in as an admin
Enter a command:
Enter an account holder:
Enter an account number belonging to the account holder:
Plan changed successfully
Enter a command:
You are now logged out
Login to start:
Enter type of account: "standard" or "admin"
Logging in as standard user - enter name:
You are now logged in as "John Doe"
Enter a command:
Enter an account number:
Enter the amount to be withdrawn:
Withdrawal of $500.00 successful
Enter a command:
You are now logged out
